data/lang/module-scoop/en.json

```diff
@@ -17,19 +17,19 @@
   },
   "ADTEXT_ARMS_DEALER": {
     "description": "",
-    "message": "PICK UP drifting container."
+    "message": "PICK UP. Prompt recovery of drifting container required."
```

Comment:
`PICK UP: ` with `:` I suspect? When fixed, I think this text improvement commit can be squashed into your scout.lua